doc/api/assert.md

```diff
@@ -665,6 +665,9 @@ assert.ok(true);
 // OK
 assert.ok(1);
 // OK
+assert.ok(typeof 123 === 'string');
+// throws "AssertionError: assert(typeof 123 === 'string')
```

Comment:
I believe this should either be updated to be `assert()` in the code or `assert.ok` in the comment.

Reply:
So that is a tricky one. The example is actually in line with the current implementation. I decided to just always show `assert` no matter what the real function name was. The reason is that it is more work to find out if the called function was `name`, `obj.name`, `obj.prop.name` and so on. It is easy to print the `name` but finding out about the potential part before the name is more work.

As a user might actually also import `assert` as any name (e.g. `foobar`), I think it is better to have `assert` as function name. But someone might of course also argue against this because of the very same reason...